docs/core/testing/unit-testing-mstest-runner-intro.md

```diff
@@ -0,0 +1,253 @@
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- All these 3 packages can be replaced by reference to:
+        <PackageReference Include="MSTest" Version="3.2.0-preview.23570.1" />
+    -->
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
+    <PackageReference Include="MSTest.TestAdapter" Version="3.2.0-preview.23570.1" />
+    <PackageReference Include="MSTest.TestFramework" Version="3.2.0-preview.23570.1" />
+
+    <PackageReference Include="coverlet.collector" Version="6.0.0" />
+  </ItemGroup>
+
+</Project>
+```
+
+Building the application will generate an executable application for your test project. For example `Contoso.MyTests.exe`.
+
+## Running & Debugging tests
+
+> [!IMPORTANT]
+> Telemetry is by default collected by the runner, see [Telemetry](./unit-testing-mstest-runner-telemetry.md) to learn more.
+
+MSTest Runner test projects are built as executables that can be run (or debugged) directly. There is no additional test running console or command that is needed.
+
+The application will exit with non-zero exit code in case of an error. For a list of known exit codes see [exit codes](unit-testing-mstest-runner-exit-codes.md).
+
+This is how you run tests in common scenarios:
+
+### In console
+
+#### Running the application directly in Console
+
+Publishing the test project using `dotnet publish` and running the application directly is another way to run your tests. For example `./Contoso.MyTests.exe`. In some scenarios it is also viable to use `dotnet build`, to produce the executable, but there can be edge cases to that for example when using Native AOT.
+
+#### Using `dotnet run`
+
+`dotnet run` can be used to build and run your test project. This is the easiest, although sometimes slow, way to run your tests. Using `dotnet run` is practical when you are editing and running tests locally because it ensures that the test project is re-built when needed. `dotnet run` will also automatically find the project in the current folder.
+
+#### Using `dotnet exec`
+
+`dotnet exec` can be used to run an already built test project, this is an alternative to running the application directly. `dotnet exec` requires path to the built test project dll.
+
+> [!IMPORTANT]
+> Providing path to the test project executable (.exe) will result in an error:
+
+> ```plaintext
+> Error:
+>   An assembly specified in the application dependencies manifest
+>   (Contoso.MyTests.deps.json) has already been found but with a different
+>   file extension:
+>     package: 'Contoso.MyTests', version: '1.0.0'
+>     path: 'Contoso.MyTests.dll'
+>     previously found assembly: 'S:\t\Contoso.MyTests\bin\Debug\net8.0\Contoso.MyTests.exe'
+> ```
+
+#### Using `dotnet test`
+
+> > [!IMPORTANT]
+> `dotnet test` is currently working only with MSTest, NUnit and xUnit tests. TPTF tests will not run using `dotnet test` and no error will be reported.
```

Comment:
I don't think we should mention TPTF as it's only developed to test the platform but won't be delivered on its own.

Reply:
```suggestion
> `dotnet test` is currently working only with MSTest, NUnit and xUnit tests.
```